app/src/main/java/ca/llamabagel/transpo/ui/search/SearchActivity.kt

```diff
@@ -38,6 +42,7 @@ class SearchActivity : AppCompatActivity() {
         val searchBar = findViewById<CustomSearchView>(R.id.search_bar)
         val recycler = findViewById<RecyclerView>(R.id.search_results_list)
 
+        viewModel.startListeningToSearchResults()
```

Comment:
This can be called in the ViewModel's `init` block.

Reply:
yea I was just checking how tests reacted